### ISSUE TITLE
require 'set' in oj adapter

### ISSUE THREAD

**Issue by yads** [intridea/multi_json#185]
The existing oj adapter uses `to_set` to determine ParseError equivalence. This is only available if you `require 'set'`.

irb output:

```
irb(main):001:0> require 'multi_json/adapters/oj'
NoMethodError: undefined method `to_set' for ["Oj::ParseError", "JSON::ParserError"]:Array
        from /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/multi_json-1.13.0/lib/multi_json/adapters/oj.rb:19:in `<class:ParseError>'
        from /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/multi_json-1.13.0/lib/multi_json/adapters/oj.rb:18:in `<class:Oj>'
        from /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/multi_json-1.13.0/lib/multi_json/adapters/oj.rb:7:in `<module:Adapters>'
        from /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/multi_json-1.13.0/lib/multi_json/adapters/oj.rb:5:in `<module:MultiJson>'
        from /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/multi_json-1.13.0/lib/multi_json/adapters/oj.rb:4:in `<top (required)>'
        from /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:133:in `require'
        from /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:133:in `rescue in require'
        from /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:40:in `require'
        from (irb):1
        from /usr/local/opt/rbenv/versions/2.4.1/bin/irb:11:in `<main>'
```

**Comment by rwz** [comment]
Oh damn, thanks for reporting. Will fix asap

**Comment by rwz** [comment]
Fixed and released 1.13.1

**Comment by yads** [comment]
Awesome, thanks for such a quick turnaround.
